blackbox/docs/admin/system-information.rst

```diff
@@ -1026,6 +1026,76 @@ For example, you can query shards like this::
   +--------+-----------+----+-----+------+---------+------+---------+---------+-------+
   SELECT 1 row in set (... sec)
 
+.. _sys-segments:
+
+Segments
+========
+
+The ``sys.segments`` table contains information about the Lucene segments
+of a shards.
+
+The segment information is useful to understand the behaviour of the underlying
+Lucene file structures for troubleshooting and performance optimization
+of shards.
+
++-------------------+-------------------------------------------+-------------+
+| Column Name       | Description                               | Return Type |
++===================+===========================================+=============+
+| ``table_schema``  | Schema name of the table of the shard.    | ``TEXT``    |
++-------------------+-------------------------------------------+-------------+
+| ``table_name``    | Table name of the shard.                  | ``TEXT``    |
++-------------------+-------------------------------------------+-------------+
+| ``node_``         | ID of the node on which the shard resides.| ``TEXT``    |
```

Comment:
pls fix as already suggested, it's `_node` and it's an object, description also is not correct.